lib/match/options.rb

```diff
@@ -91,7 +91,28 @@ def self.available_options
                                          end
                                        end
                                      end,
-                                     optional: true)
+                                     optional: true),
+        FastlaneCore::ConfigItem.new(key: :mobile_provision,
+                                     env_name: "MATCH_MOBILE_PROVISION",
+                                     description: "Path to the mobile provision to save - only used in manual mode",
+                                     is_string: true,
+                                     short_option: '-m',
+                                     optional: true,
+                                     default_value: nil),
+        FastlaneCore::ConfigItem.new(key: :cert,
+                                     env_name: "MATCH_CERT",
+                                     description: "Path to the certificate to save - only used in manual mode",
+                                     is_string: true,
+                                     short_option: '-c',
+                                     optional: true,
+                                     default_value: nil),
+        FastlaneCore::ConfigItem.new(key: :p12,
+                                     env_name: "MATCH_P_12",
+                                     description: "Path to the private key to save - only used in manual mode",
+                                     short_option: '-p',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.